arrow-core-data/src/test/kotlin/arrow/core/ListKTest.kt

```diff
@@ -73,6 +74,7 @@ class ListKTest : UnitSpec() {
         this::bijection),
       MonoidKLaws.laws(ListK.monoidK(), ListK.genK(), ListK.eqK()),
       TraverseLaws.laws(ListK.traverse(), ListK.genK(), ListK.eqK()),
+      FoldableLaws.laws(ListK.foldable(), ListK.genK()),
```

Comment:
Isn't this already run through `TraverseLaws`?

Reply:
Yes, my fault!